projects/common/src/index.ts

```diff
@@ -175,14 +175,17 @@ export interface HTMLElement {
     id: 'html'
     html: string
 }
+
+type ImageRoles = 'supporting' | 'immersive' | 'showcase' | 'thumbnail' | string
```

Comment:
I added this to get autocomplete hints, not sure if its a good idea or not? Can remove

Reply:
I'm not sure if there are more types (thats why I added 'string' in the end)